colony/shell.py

```diff
@@ -18,15 +18,15 @@
     sb, sandbox         start sandbox, end sandbox and get its status

```

Comment:
add 'configure' command to main usage message

Reply:
done